### DISCUSSION TITLE
Is there a way to change BUILDKIT_STEP_LOG_MAX_SIZE and BUILDKIT_STEP_LOG_MAX_SPEED variables?

### DISCUSSION THREAD

**Discussion by Noksa** [rancher-sandbox/rancher-desktop#4088] [General]
Please refer to the same question here: https://github.com/docker/for-mac/issues/6332

How to do that in Rancher Desktop?

I have the same problem:

<img width="980" alt="image" src="https://user-images.githubusercontent.com/36312857/222697841-43bb68f3-c12b-4da6-af7f-953d04606068.png">

**Comment by adamkpickering** [comment]
As far as I can tell, the BUILDKIT_STEP_LOG_MAX_SIZE and BUILDKIT_STEP_LOG_MAX_SPEED environment variables must be set for `dockerd`, not the docker CLI.

If this is the case, you can do this by modifying the environment of the Lima VM in which `dockerd` runs. You can get into the VM with `rdctl shell`. From there, I'm a little less sure of what to do - adding the necessary environment variables to `/etc/profile` might do the trick. It is probably possible to set these environment variables for `dockerd` only by modifying the openrc service, or the openrc service config, for `dockerd`.

You might also consider modifying `~/Library/Application\ Support/rancher-desktop/lima/_config/override.yaml` so that it runs a script when the VM starts up. You may find the [Lima default config](https://github.com/lima-vm/lima/blob/master/examples/default.yaml) helpful for this.

Note that anything you do in the VM may be wiped away when you restart RD, and it will definitely be wiped away when you do a factory reset.

@jandubois I you're more familiar with modifications to the lima VM - do you have any better ideas?
